transport.go

```diff
@@ -25,16 +25,18 @@ import (
 	"time"
 )
 
-// This default transport is similar to http.DefaultTransport
-// but with additional DisableCompression:
-var defaultMinioTransport http.RoundTripper = &http.Transport{
+// DefaultTransport - this default transport is similar to
+// http.DefaultTransport but with additional param  DisableCompression
+// is set to true to avoid decompressing content with 'gzip' encoding.
+var DefaultTransport http.RoundTripper = &http.Transport{
 	Proxy: http.ProxyFromEnvironment,
 	DialContext: (&net.Dialer{
 		Timeout:   30 * time.Second,
 		KeepAlive: 30 * time.Second,
 		DualStack: true,
 	}).DialContext,
 	MaxIdleConns:          100,
+	MaxIdleConnsPerHost:   100,
```

Comment:
When you have `MaxIdleConnsPerHost`, do you still need `MaxIdleConns`?

Reply:
yes we need @ebozduman see the original issue for more explanation. http://tleyden.github.io/blog/2016/11/21/tuning-the-go-http-client-library-for-load-testing/